controllers/src/main/scala/cromwell/pipeline/controller/ProjectController.scala

```diff
@@ -3,17 +3,29 @@ package cromwell.pipeline.controller
 import akka.http.scaladsl.model.StatusCodes
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server.Route
+import cromwell.pipeline.controller.utils.PathMatchers.ProjectId
 import cromwell.pipeline.datastorage.dto.auth.AccessTokenContent
 import cromwell.pipeline.datastorage.dto.{ ProjectAdditionRequest, ProjectUpdateNameRequest }
 import cromwell.pipeline.service.ProjectService.Exceptions.{ ProjectAccessDeniedException, ProjectNotFoundException }
 import cromwell.pipeline.service.{ ProjectService, VersioningException }
-import cromwell.pipeline.controller.utils.PathMatchers.ProjectId
 import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+
 import scala.util.{ Failure, Success }
 
 class ProjectController(projectService: ProjectService) {
 
-  private def getProject(implicit accessToken: AccessTokenContent): Route = get {
+  private def getProject(implicit accessToken: AccessTokenContent): Route = path(ProjectId) { projectId =>
+    get {
```

Comment:
Please change get and pah between their places.

Reply:
Done